### PR TITLE
Feature: view donation form link

### PR DESCRIPTION
**Pull request by alaca**
## Description

This PR adds the view donation form link/icon.  

## Visuals

Screen Capture_select-area_20231013182519

## Pre-review Checklist

-   [x] Acceptance criteria satisfied and marked in related issue
-   [ ] Relevant `@unreleased` tags included in DocBlocks
-   [ ] Includes unit tests
-   [ ] Reviewed by the designer (if follows a design)
-   [x] [Self Review](https://give.gitbook.io/development-manual/devops/github/code-reviews#self-review) of code and UX completed

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205720210343673

### CODE REVIEW

File: src/FormBuilder/resources/js/form-builder/src/containers/HeaderContainer.tsx [hunk 1]
```diff
@@ -142,6 +156,11 @@ const HeaderContainer = ({
                                     ? __('Save as Draft', 'give')
                                     : __('Switch to Draft', 'give')}
                         </Button>
+                        <Button
+                            label={__('View form', 'give')}
+                            onClick={() => window.location.href = permalink}
+                            icon={external}
+                        />
```
Comment (JasonTheAdams) [inline src/FormBuilder/resources/js/form-builder/src/containers/HeaderContainer.tsx:163]: Let's make this a proper anchor for a11y:
```suggestion
                        <Button
                            label={__('View form', 'give')}
                            href={permalink}
                            target="_blank"
                            icon={external}
                        />
```

**Comment by JoshuaHungDinh** [comment]
@alaca, This works for both published and draft for me - just the one small oddity. 

https://github.com/impress-org/givewp/assets/75056371/b831afa2-1335-4ea6-ae43-d74cfce7ba3e

**Comment by JasonTheAdams** [comment]
@JoshuaHungDinh Mind giving this another review?

**Review by JoshuaHungDinh — APPROVED** [review]
Tested and working! 

This is good to go. @JasonTheAdams

**Review by Benunc — APPROVED** [review]
This looks amazing. This just saved us 100 hours of support and customer success time, guaranteed. Little fix, but a HUGE improvement.

**Pull request merged**